<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKUpdateManager</class>
 <widget class="QDialog" name="SAKUpdateManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check for update</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="6">
    <widget class="QWidget" name="widget8888" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="visitWebPushButton">
        <property name="text">
         <string>ReleaseSite</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="checkForUpdatePushButton">
        <property name="text">
         <string>CheckForUpdate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="currentVersionLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#currentVersionLabel{
	color:blue
}</string>
     </property>
     <property name="text">
      <string notr="true">0.0.0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="6">
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Packet list</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QListWidget" name="downloadListListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QLabel" name="noNewVersionTipLabel">
     <property name="styleSheet">
      <string notr="true">QLabel#noNewVersionTipLabel{
	color:blue
}</string>
     </property>
     <property name="text">
      <string>Has no version to update</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color:red
}</string>
     </property>
     <property name="text">
      <string>Update function is not stable, please visit the release web site to get the latest version from &quot;Links&quot; menu.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Current version</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2" colspan="5">
    <widget class="QProgressBar" name="updateProgressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QLabel" name="updateProgressLabel">
     <property name="text">
      <string>Check for update progress</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="newVersionLabel">
     <property name="styleSheet">
      <string notr="true">QLabel#newVersionLabel{
	color:red
}</string>
     </property>
     <property name="text">
      <string notr="true">(0.0.0)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Newer version</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
